pkg/objectstate/rte/rte.go

```diff
@@ -180,21 +179,27 @@ func (em *ExistingManifests) State(mf rtemanifests.Manifests, instance *nropv1al
 			},
 		)
 
-		if mf.ConfigMap != nil {
-			desiredConfigMap := mf.ConfigMap.DeepCopy()
-			desiredConfigMap.Name = generatedName
+		if mf.MachineConfig != nil {
+			if mcp.Spec.MachineConfigSelector == nil {
+				klog.Warningf("the machine config pool %q does not have machine config selector", mcp.Name)
+				continue
+			}
+			desiredMachineConfig := mf.MachineConfig.DeepCopy()
+			// prefix machine config name to guarantee that we will have an option to override it
+			desiredMachineConfig.Name = fmt.Sprintf("51-%s", generatedName)
```

Comment:
Can you use https://github.com/openshift-kni/numaresources-operator/blob/1ff7e5148afe0b3b8ca0ffc33e227818ad17f125/pkg/objectstate/rte/rte.go#L334?

Reply:
This was actually a rebase glitch, removed